tests/MassTransit.Azure.Cosmos.Tests/MissingInstance_Specs.cs

```diff
@@ -147,8 +147,8 @@ public async Task Should_work_as_expected()
                         x.AddSagaStateMachine<MissingInstanceStateMachine, MissingInstance>()
                             .CosmosRepository(r =>
                             {
-                                r.EndpointUri = Configuration.EndpointUri;
-                                r.Key = Configuration.Key;
+                                r.AccountEndpoint = Configuration.AccountEndpoint;
```

Comment:
This is a breaking change, that will prevent people from upgrading without forced changes to figure out what is different. Retain the existing properties with `[Obsolete("Use NewPropertyName")]` if you change existing names.

Reply:
Marked these as obsolete in favor of the new names that are more in line with the Cosmos v3 library.

ICosmosSagaRepositoryConfigurator:
* Key -> AuthKeyOrResourceToken
* EndpointUri -> AccountEndpoint

AzureCosmosEmulatorConstants:
* Key -> AccountKey
* EndpointUri -> AccountEndpoint